libtenzir/src/importer.cpp

```diff
@@ -206,6 +213,16 @@ importer(importer_actor::stateful_pointer<importer_state> self,
     self->state.index = std::move(index);
     self->state.stage->add_outbound_path(self->state.index);
   }
+  for (const auto& plugin : plugins::get<analyzer_plugin>()) {
+    // We can safely assert that the analyzer was already initialized. The
+    // pipeline API guarantees that remote operators run after the node was
+    // successfully initialized, which implies that analyzers have been
+    // initialized as well.
+    // TODO: Is this actually true?
```

Comment:
I think so. The node itself does not allow any incoming messages before initializing components.